src/pages/objkt-display/index.js

```diff
@@ -156,14 +163,13 @@ export const ObjktDisplay = () => {
             <Container>
               <Padding>
                 {restricted && (
-                  <div
-                    style={{
-                      color: 'white',
-                      background: 'black',
-                      textAlign: 'center',
-                    }}
-                  >
-                    Restricted OBJKT
+                  <div className={styles.restricted}>
+                    Restricted OBJKT. Contact the Teia moderators on <a href="https://discord.gg/TKeybhYhNe" target="_blank" rel="noreferrer">Discord</a> to resolve the status. See the <a href="https://github.com/teia-community/teia-docs/wiki/Core-Values-Code-of-Conduct-Terms-and-Conditions#3-terms-and-conditions---account-restrictions" target="_blank" rel="noreferrer">Teia Terms and Conditions</a>.
```

Comment:
Might be done in a separate PR, but if possible it would be better to use an "external" file for texts

Reply:
Agreed. Seems like a lot of text in the GUI at the moment.